nemo/collections/asr/losses.py

```diff
@@ -58,3 +58,42 @@ def _loss(self, log_probs, targets, input_length, target_length):
 
     def _loss_function(self, **kwargs):
         return self._loss(*(kwargs.values()))
+
+
+class CrossEntropyLossNM(LossNM):
```

Comment:
could you reuse this: https://github.com/NVIDIA/NeMo/blob/master/nemo/backends/pytorch/common/losses.py#L99

Reply:
Woops. Didn't see this one before. Will revert my version.

Reply:
Didn't see this before. I'll revert my version.

Reply:
Since that is a backend function, should I extend the above class and just add `reduction="mean"` as an argument? Would be a much cleaner implementation.

Reply:
I would just modify this one to be more flexible.

Reply:
I added the reduction kwarg to the backend function and aliased it in asr collection so that it can be used as part of the collection.